package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "fantasy-land": "3.5.x",
     "jsverify": "0.8.x",
-    "sanctuary-type-classes": "8.1.x"
+    "sanctuary-type-classes": "8.1.1"
```

Comment:
I considered `"~8.1.1"` here, but these projects are so tightly coupled that specifying an exact version seems prudent.